src/pandas_profiling/config_default.yaml

```diff
@@ -77,25 +77,39 @@ missing_diagrams:
 correlations:
     pearson:
       calculate: True
+      spark_calculate: True
```

Comment:
I wouldn't introduce calculate parameters for each engine (pandas/spark). Rather have a single flag for engine. For spark some correlations won't be implemented, for which we can ignore the configuration alltogether.

Reply:
spark_calculate removed as config param